README.rst

```diff
@@ -37,9 +37,17 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy, please see NeuroStars:
```

Comment:
Sure - github issues if it may be a bug, mailing list for general questions or advice about raising an issue.